src/loaders/lottie/tvgLottieBuilder.cpp

```diff
@@ -929,6 +929,20 @@ static void _fontText(TextDocument& doc, Scene* scene)
 }
 
 
+static void _rangeColor(LottieColor& color, RGB24& baseColor, float frameNo, Tween& tween, LottieExpressions* exps, float factor)
```

Comment:
I think like this would be a little bit simpler?

```
static void _rangeColor(const RGB24& rangeColor, RGB24& color, float factor)
{
    if (tvg::equal(factor, 1.0f)) {
        color = rangeColor;
        return;
    }

    color.rgb[0] = tvg::lerp<uint8_t>(color.rgb[0], rangeColor.rgb[0], factor);
    color.rgb[1] = tvg::lerp<uint8_t>(color.rgb[1], rangeColor.rgb[1], factor);
    color.rgb[2] = tvg::lerp<uint8_t>(color.rgb[2], rangeColor.rgb[2], factor);
}
```